pip/client/k8s.go

```diff
@@ -0,0 +1,67 @@
+package client
```

Comment:
The discovery code looks very general - i.e. not pip client specific. Would you consider making this a separate package so it can potentially be reused elsewhere?

Reply:
Then it's better to put it in some other github.com/infobloxopen repo.